### PR TITLE
Fix for relative paths

### PR DESCRIPTION
**Pull request by hoffmabc**

### CODE REVIEW

File: js/utils/loadTemplate.js [hunk 1]
```diff
@@ -3,7 +3,11 @@ var fs = require('fs');
 
 module.exports = function(templateFile, callback){
 
-  fs.readFile(templateFile, "utf8", function(err, ob){
+  var loc = window.location.pathname;
+  var cur_dir = loc.substring(0, loc.lastIndexOf('/'));
+
+  fs.readFile(cur_dir + '/' + templateFile, "utf8", function(err, ob){
```
Comment (gubatron) [inline js/utils/loadTemplate.js]: Q: does `fs.readFile` work ok with forward path '/' separators in windows?
(I'm a noob with node, in every other language there's usually a path separator object to guarantee cross os compatibility)

Comment (gubatron) [inline js/utils/loadTemplate.js]: A: looks like you gotta use [`path.sep`](https://nodejs.org/api/path.html#path_path_sep) @hoffmabc  (available via `require('path')`)

Might also want to search for it on line 7.

```
var cur_dir = loc.substring(0, loc.lastIndexOf(path.sep));
```

Comment (hoffmabc) [inline js/utils/loadTemplate.js]: I don't think this is going to hold up anyways so I will be submitting a fix. Found some permission issues with approaching this in this way.

Comment (hoffmabc) [inline js/utils/loadTemplate.js]: I'll take a look.

Comment (jjeffryes) [inline js/utils/loadTemplate.js]: Here's some documentation on it http://shapeshed.com/writing-cross-platform-node/

It looks like path.join would put in the right separator.

**Comment by hoffmabc** [comment]
Can we merge this @morebrownies @jjeffryes ?

**Comment by jjeffryes** [comment]
I'll take a look at it once I finish my current branch.

**Pull request merged**